web/src/services/permission.ts

```diff
@@ -0,0 +1,9 @@
+import http from "./http";
+
+var permission = {
```

Comment:
Pluralize this please

Reply:
[copy](https://github.com/zinclabs/zinc/blob/main/web/src/services/user.ts#L3)

Reply:
[copy](https://github.com/zinclabs/zinc/blob/main/web/src/services/user.ts#L3)